product_docs/docs/epas/15/epas_compat_spl/10_collections/index.mdx

```diff
@@ -24,6 +24,28 @@ The general steps for using a collection are the following:
 -   To initialize a null collection, you must either make it an empty collection or assign a non-null value to it. Generally, a null collection is initialized by using its *constructor*.
 -   To add elements to an empty associative array, you can assign values to its keys. For nested tables and varrays, generally its constructor is used to assign initial values to the nested table or varray. For nested tables and varrays, you then use the `EXTEND` method to grow the collection beyond its initial size set by the constructor.
 
+
+## Limitations
+
+- Multilevel collections (that is, where the data item of a collection is another collection) aren't supported.
+
+- Columns of collection types aren't supported.
+
+  This example shows the error displayed when we try to create the column of collection type:
+
+  ```sql
+  --Use varray in composite type and then use it to create table 
```

Comment:
You aren't creating the table in the lines within the scope of this comment, are you?
```suggestion
  # Use varray in composite type, footype 
```